Lib/test/_test_multiprocessing.py

```diff
@@ -6126,7 +6126,8 @@ class ThreadsMixin(BaseMixin):
 # Functions used to create test cases from the base ones in this module
 #
 
-def install_tests_in_module_dict(remote_globs, start_method):
+def install_tests_in_module_dict(remote_globs, start_method,
+                                 only_type=None, exclude_types=False):
```

Comment:
This function needs proper documentation as to what it does and how it behaves. The new parameters behaviors are unobvious.